azure-pipelines.yml

```diff
@@ -1,4 +1,12 @@
+# By default, ADO runs jobs on pushes and on PRs. Since we're only tracking/after PRs, turn off 
+# tiggers on push.
 trigger:
+  branches:
+    exclude:
```

Comment:
It is not _100%_ clear to me that we actually want to disable this?

It might break something like `/azp run` I am not sure... It was enabled manually in the backend only for master, so if folks prefer I can leave that configuration here.